config/database.yml

```diff
@@ -64,8 +64,8 @@ test:
   <<: *default
   database: hunterskeepers_test
   host: <%= ENV['DB_HOST'] || 'localhost' %>
-  username: <%= ENV['POSTGRES_USER'] || 'root' %>
-  password: <%= ENV['POSTGRES_PASSWORD'] || nil %>
+  username: <%= ENV['POSTGRES_USER'] || "itox" %>
+  password: <%= ENV['POSTGRES_PASSWORD'] || "itox" %>
```

Comment:
Hey, looks like these are still in here!

Reply:
it's done